### ISSUE TITLE
New Game Idea?

### ISSUE THREAD

**Issue by aldy-san** [aldy-san/zero-neko#5]
If you have any game idea about learning japanese you can make issue or just write your idea in [GAME_IDEA.md](https://github.com/aldy-san/zero-neko/blob/master/GAME_IDEA.md)

**Comment by nayan2306** [comment]
Truth and dare only gaming app.

**Comment by aldy-san** [comment]
great, but can you write simple desc about that at [GAME_IDEA.md](https://github.com/aldy-san/zero-neko/blob/master/GAME_IDEA.md)

**Comment by PalakJain9** [comment]
Hey, I'd like to suggest a memory game idea that will help people learn and practice Japanese. It works on the idea that people have to match words in the scenario of memory game( 5x5 tile ).
Kindly assign me this issue :)

**Comment by nayan2306** [comment]
Yes sure 

> great, but can you write simple desc about that at [GAME_IDEA.md](https://github.com/aldy-san/zero-neko/blob/master/GAME_IDEA.md)

**Comment by aldy-san** [comment]
@PalakJain9 Sure you can, thanks.
@nayan2306 Great, Thanks.

**Comment by Specy** [comment]
a typing game where u learn how to write faster interactively, i made something like that a while ago for school homework, check:
https://specy.altervista.org/apps/p3/words/index.html at "impara la tastiera"

**Comment by aldy-san** [comment]
@Specy i think its good game idea. you can write simple description about that at [GAME_IDEA.md](https://github.com/aldy-san/zero-neko/blob/master/GAME_IDEA.md), if you want. thanks.

**Comment by Specy** [comment]
@aldy-san i saw that there is already something similar to what i said in the code, called speedracer, are you throwing that out?

**Comment by aldy-san** [comment]
@Specy yeah, but i think, i can modify your game idea to make new game, later.

**Comment by nayan2306** [comment]
@aldy-san I have upadate from my side already anything you want from my side just ping me

**Comment by Specy** [comment]
@aldy-san i can make the game myself, just wanted to know if it was gonna be used if i make it, or if there is one already so i shouldn't

**Comment by aldy-san** [comment]
@nayan2306 okay, thanks.
@Specy sure, it will be used if you make it.
